src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -600,6 +600,28 @@ internal Notification CreateNotification(NotificationVersionOne v1Notification)
             return notification;
         }
 
+        private List<Location> CreateLocations(PhysicalLocationVersionOne v1PhysicalLocation)
+        {
+            List<Location> locations = null;
+
+            if (v1PhysicalLocation != null)
+            {
+                locations = new List<Location>
+                {
+                    new Location
+                    {
+                        PhysicalLocation = new PhysicalLocation
+                        {
+                            ArtifactLocation = CreateFileLocation(v1PhysicalLocation),
+                            Region = CreateRegion(v1PhysicalLocation.Region)
+                        }
```

Comment:
Why not continue to use existing `CreatePhysicalLocation` here?
```
new Location
{
    PhysicalLocation = CreatePhysicalLocation(v1PhysicalLocation)
}
```

Reply:
thanks!

---
In reply to: [278290863](https://github.com/Microsoft/sarif-sdk/pull/1456#discussion_r278290863) [](ancestors = 278290863)